x-pack/legacy/plugins/siem/public/components/events_viewer/events_viewer.tsx

```diff
@@ -144,7 +144,10 @@ export const EventsViewer = React.memo<Props>(
                         totalCount={totalCount}
                       />
 
-                      <div data-test-subj="events-container" style={{ width: `${width}px` }}>
+                      <div
+                        data-test-subj={`events-container-${loading}`}
```

Comment:
When inspected in Chrome, the `data-test-subj` of this `div` reads:

```
data-test-subj="events-container-false"
```

Since `false` represents the loading state, (as opposed to a particular id), would you find a format like this:

```
data-test-subj={`events-container-loading-${loading}`}
```

...which would in-turn look like this in the DOM when inspected:

```
data-test-subj="events-container-loading-false"
```

...easier to grok at-a-glance?

Reply:
(the suggested pattern would also be applicable to other tests that currently use the `thing-${loading}` `data-test-subj` pattern)

Reply:
sure im good w this